src/eventbridge/index.ts

```diff
@@ -0,0 +1,78 @@
+import { aws_events } from "aws-cdk-lib";
```

Comment:
Can we adopt the code tenet of reserving `index.its` for `export * from "./file"` instead of putting logic in there?

Reply:
Sure.